lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -4,64 +4,68 @@
 from typing import (
     cast,
     Optional,
+    Type,
+    TypeVar,
 )
 
 from fastapi import (
     Cookie,
-    Depends,
     Header,
     Query,
 )
-from sqlalchemy.orm import Session
+from fastapi.params import Depends
+from starlette.requests import Request
```

Comment:
I suppose that if we directly import this, `starlette` needs to be added to the `webapp` package and (together with `lagom`) to `pyproject.toml`.